resources/dicts/events/ceremonies/ceremony-master.json

```diff
@@ -2937,5 +3306,86 @@
       "all_traits"
     ],
     "Time has taken its toll and m_c has noticed {PRONOUN/m_c/self} slowing down. {PRONOUN/m_c/subject/CAP} {VERB/m_c/have/has} worked timelessly for many moons, but it is time for {PRONOUN/m_c/object} to retire. "
+  ],
+  "elder_6": [
+    [
+      "elder",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "all_traits"
+    ],
+    "m_c holds {PRONOUN/m_c/poss} head up high as l_n calls upon StarClan to witness m_c's service to c_n. This is a retirement hard earned."
+  ],
+  "elder_7": [
+    [
+      "elder",
+      "general_mentor",
+      "general_parents",
+      "general_leader",
+      "general_backstory",
+      "all_traits"
+    ],
+    "As m_c entering into the sunset of {PRONOUN/m_c/poss} life, {PRONOUN/m_c/subject} {VERB/m_c/join/joins} the ranks of the elders without regret, looking forward to the rest {PRONOUN/m_c/subject} {VERB/m_c/have/has} earned."
```

Comment:
"as m_c entering into the sunset..." should be written as "as m_c enters into the sunset..." or something similar